Stream/DynamicDestinationMessaging/Program.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+using Steeltoe.Stream.Extensions;
+
+namespace DynamicDestinationMessaging
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>();
+                })
+                .AddStreamServices<Program>();
```

Comment:
This is not being really used, since there is no EnableBinding attribute specified. I do think we need to demonstrate an ISink at least.